examples/sunlinsol/lapackband/test_sunlinsol_lapackband.c

```diff
@@ -135,13 +135,12 @@ int main(int argc, char *argv[])
   /* Print result */
   if (fails) {
     printf("FAIL: SUNLinSol module failed %i tests \n \n", fails);
-    printf("\nA (original) =\n");
-    SUNBandMatrix_Print(B,stdout);
```

Comment:
Was printing of the banded matrices causing the test failures, or were these removed for some other reason?

Reply:
It was not contributing to the failures, but it was making deciphering the log difficult.